Fix pointers to wrong documentation files.

solved upstream, r114649.

 -- Victor Seva <linuxmaniac@torreviejawireless.org>

--- a/configs/iax.conf.sample
+++ b/configs/iax.conf.sample
@@ -198,7 +198,7 @@ forcejitterbuffer=no
 ;
 ;authdebug=no
 ;
-; See doc/README.tos for a description of the tos parameters.
+; See doc/ip-tos.txt for a description of the tos parameters.
 ;tos=ef
 ;
 ; If regcontext is specified, Asterisk will dynamically create and destroy
--- a/configs/iaxprov.conf.sample
+++ b/configs/iaxprov.conf.sample
@@ -53,7 +53,7 @@ codec=ulaw
 ;
 flags=register,heartbeat
 ;
-; See doc/README.tos for a description of this parameter.
+; See doc/ip-tos.txt for a description of this parameter.
 ;tos=ef
 ;
 ; Example iaxy provisioning
--- a/configs/sip.conf.sample
+++ b/configs/sip.conf.sample
@@ -53,7 +53,7 @@ srvlookup=yes			; Enable DNS SRV lookups
 				; and multiline formatted headers for strict
 				; SIP compatibility (defaults to "no")
 
-; See doc/README.tos for a description of these parameters.
+; See doc/ip-tos.txt for a description of these parameters.
 ;tos_sip=cs3                    ; Sets TOS for SIP packets.
 ;tos_audio=ef                   ; Sets TOS for RTP audio packets.
 ;tos_video=af41                 ; Sets TOS for RTP video packets.
@@ -567,7 +567,7 @@ srvlookup=yes			; Enable DNS SRV lookups
 ;allow=g723.1			; Asterisk only supports g723.1 pass-thru!
 ;allow=g729			; Pass-thru only unless g729 license obtained
 ;callingpres=allowed_passed_screen	; Set caller ID presentation
-				; See README.callingpres for more information
+				; See doc/callingpres.txt for more information
 
 
 ;[xlite1]
--- a/configs/zapata.conf.sample
+++ b/configs/zapata.conf.sample
@@ -276,7 +276,7 @@ callwaiting=yes
 ;
 ; Whether or not use the caller ID presentation for the outgoing call that the
 ; calling switch is sending.
-; See README.callingpres
+; See doc/callingpres.txt
 ;
 usecallingpres=yes
 ;
